src/views/historical/testing-gap/dashboard.vue

```diff
@@ -0,0 +1,385 @@
+<template>
+  <DashboardCard width="full">
+    <template #subtitle>
+      TThis tool shows us places where fewer people were tested for COVID-19
```

Comment:
Noticed this typo but I assume that we're not really taking a look at this this go around though

```suggestion
      This tool shows us places where fewer people were tested for COVID-19
```